Tools/Scripts/libraries/webkitcorepy/webkitcorepy/task_pool.py

```diff
@@ -264,7 +296,7 @@ def handler(cls, value, _):
             cls.working = False
 
     @classmethod
-    def main(cls, name, loglevel, setup, setupargs, setupkwargs, queue, teardown, teardownargs, teardownkwargs):
+    def main(cls, name, loglevel, setup, setupargs, setupkwargs, queue, teardown, teardownargs, teardownkwargs, mutually_exclusive_groups):
```

Comment:
It's slightly surprising to me that this is a dict here, versus a list everywhere else. Not sure it makes sense to change anything however?

Reply:
I'll change this to `mutuals_exclusive_group_queues`, it's an internal argument anyways.